chart/openfaas/templates/gateway-dep.yaml

```diff
@@ -227,12 +227,18 @@ spec:
             value: "{{ .Values.faasnetes.readinessProbe.timeoutSeconds }}"
           - name: readiness_probe_period_seconds
             value: "{{ .Values.faasnetes.readinessProbe.periodSeconds }}"
+          - name: readiness_probe_success_threshold
+            value: "{{ .Values.faasnetes.readinessProbe.successThreshold }}"
```

Comment:
If someone has a saved values.yaml - or a complete copy of values.yaml, and they install - the value will be null probably or empty string? What will happen?

Reply:
For the missing values in the custom `values.yaml` file the defaults from the charts `values.yaml` file will be used.